### PR TITLE
Add a link to Mastodon and some emojis

### PR DESCRIPTION
**Pull request by benjaoming**
This one goes out to you @humitos :heart:

### CODE REVIEW

File: _themes/rtd-blog/social.html [hunk 1]
```diff
@@ -56,10 +56,13 @@ <h3> Follow us </h3>
 
 <ul>
 <li>
-<a href="http://blog.readthedocs.com/archive/atom.xml" rel="me">Atom Feed</a>
+<a href="http://blog.readthedocs.com/archive/atom.xml" rel="me">Atom Feed 📶️</a>
 </li>
 <li>
-<a href="https://twitter.com/readthedocs" rel="me">Twitter</a>
+<a href="https://fosstodon.org/@readthedocs/" rel="me">Mastodon 🐘️</a>
```
Comment (humitos) [inline _themes/rtd-blog/social.html:62]:
```suggestion
<a href="https://fosstodon.org/@readthedocs" rel="me">Mastodon 🐘️</a>
```

File: _themes/rtd-blog/social.html [hunk 2]
```diff
@@ -56,10 +56,13 @@ <h3> Follow us </h3>
 
 <ul>
 <li>
-<a href="http://blog.readthedocs.com/archive/atom.xml" rel="me">Atom Feed</a>
+<a href="http://blog.readthedocs.com/archive/atom.xml" rel="me">Atom Feed 📶️</a>
```
Comment (humitos) [inline _themes/rtd-blog/social.html:59]:
```suggestion
<a href="http://blog.readthedocs.com/archive/atom.xml" rel="me">📶️ Atom Feed</a>
```

I would prefix all the texts with the emojis so they are aligned.

**Review by humitos — APPROVED** [review]

File: _themes/rtd-blog/social.html (hunk 2 above)
Comment (benjaoming) [reply _themes/rtd-blog/social.html:59]: I do usually prefer that but for some reason the other way looked better (to me)

image

File: _themes/rtd-blog/social.html (hunk 1 above)
Comment (benjaoming) [reply _themes/rtd-blog/social.html:62]: Ah yes, without the slash is normal. I must have hit some kind of random error, I tried adding the slash and then the error went away :shrug:

**Pull request merged**